_includes/v22.2/ui/active-statement-executions.md

```diff
@@ -0,0 +1,38 @@
+
+## Active Executions table
+
+Click <img src="{{ 'images/common/ui-columns-button.png' | relative_url }}" alt="Column selector" /> to select the columns to display in the table.
+
+Column | Description
+-----|------------
+Statement Execution ID | The execution ID of the statement execution.
+Statement Execution | The SQL statement that was executed.
+Status | The status of the execution: `Preparing`, `Waiting`, or `Executing`.
+Start Time | The timestamp the execution started.
```

Comment:
Suggesting for clarity, as this could read as "a timestamp started by the execution"

```suggestion
Start Time | The timestamp when the execution started.
```

Reply:
Fixed.